dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -19,7 +19,7 @@ class Api::V1::SectionsControllerTest < ActionController::TestCase
     @section_with_script = create(:section, user: @teacher, script: Script.flappy_script)
     @student_with_script = create(:follower, section: @section_with_script).student_user
 
-    @csp_course = create(:course, name: 'csp-2017')
+    @csp_course = create(:course, name: 'csp-2017', visible: true, is_stable: true)
```

Comment:
should there be a csp course that is visible and not stable as well to test cases with that?

Reply:
good idea